public/js/front.js

```diff
@@ -470,12 +462,20 @@ const updateCart = (id, quantity) => {
         dataToSend,
         function(data, status) {
             // alert("Data: " + data + "\nStatus: " + status);
+            function notify(type, title, text) {
+                new PNotify.alert({
+                    title: title,
+                    text: text,
+                    type: type
+                });
+            }
             notify('info', 'Successful', ' Cart has been Updated')
         });
 
 }
 
 const deleteCart = (id, quantity, price) => {
+    notify()
     const dataTodelete = {
         id,
         quantity,
```

Comment:
quantity and price are not needed here, only the id is needed is needed to delete an item from the cart on the server. Check